vendor/src/github.com/docker/notary/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
There was previously a bug in the vendoring script which stripped Licenses, it was merged after the initial vendoring commit.